dev/benchmarks/complex_layout/android/build.gradle

```diff
@@ -5,7 +5,7 @@ buildscript {
     }
 
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.2.1'
+        classpath 'com.android.tools.build:gradle:3.4.2'
```

Comment:
For this and the gradle version - should we roll to latest?  If not, can you update the PR description to explain why we're not and whatever is blocking us from doing it?